test/benchmarks/material/button-harness/BUILD.bazel

```diff
@@ -0,0 +1,82 @@
+load("@npm_angular_dev_infra_private//benchmark/component_benchmark:component_benchmark.bzl", "component_benchmark")
+load("//tools:defaults.bzl", "ng_test_library", "ng_web_test_suite", "ts_library")
+
+package(default_visibility = ["//visibility:public"])
+
+# These are two separate files that provide the same tidy interface for running a performance
+# benchmark.
+#
+# I am not sure why the testbed version of the test does not log console.time to the terminal,
+# although I assume it has something to do with the default karma config that is generated by
+# karma_web_test_suite (which is wrapped by ng_web_test_suite).
+#
+# I am equally lost as to why the performance api exists for the testbed tests but not for the
+# protractor tests.
```

Comment:
What performance API? you mean `console.time`?

Reply:
Ah, makes sense. Yeah, we should probably use `console.time` in Node

Reply:
Actually, looks like in browsers we should use `performance.now` and in Node `process.hrtime`

Reply:
Hmm, I'll try this out. It would allow me to average test times for protractor tests